x-pack/plugin/redact/src/main/java/org/elasticsearch/xpack/redact/RedactProcessor.java

```diff
@@ -203,6 +215,20 @@ private static void matchRepeat(Grok grok, byte[] utf8Bytes, RegionTrackingMatch
         } while (offset != length);
     }
 
+    private void updateMetadataIfNecessary(IngestDocument ingestDocument, String fieldValue, String redacted) {
+        if (traceRedact == false) return;
+        if (fieldValue == null) return;
```

Comment:
Style nitpick: These conditionals could be combined, but more importantly I think it's best to always put the braces in.